python_modules/dagster/dagster_tests/storage_tests/test_asset_value_loader.py

```diff
@@ -216,3 +216,66 @@ def repo():
         with repo.get_asset_value_loader() as loader:
             value = loader.load_asset_value(AssetKey("asset1"), partition_key="2020-05-05")
             assert value == 5
+
+
+def test_io_manager_with_config():
+    class MyIOManager(IOManager):
+        def __init__(self, key):
+            self.key = key
+
+        def handle_output(self, context, obj):
+            assert False
+
+        def load_input(self, context):
+            return self.key
+
+    @io_manager(config_schema={"key": int})
+    def my_io_manager(context):
+        return MyIOManager(context.resource_config["key"])
+
+    @asset
+    def asset1():
+        ...
+
+    @repository
+    def repo():
+        return with_resources([asset1], resource_defs={"io_manager": my_io_manager})
+
+    resource_config = {"io_manager": {"config": {"key": 5}}}
+    with repo.get_asset_value_loader() as loader:
+        value = loader.load_asset_value(AssetKey("asset1"), resource_config=resource_config)
+        assert value == 5
+
+
+def test_io_manager_resource_with_config():
+    @resource(config_schema={"key": int})
+    def io_resource(context):
+
+        return context.resource_config["key"]
+
+    class MyIOManager(IOManager):
+        def handle_output(self, context, obj):
+            assert False
+
+        def load_input(self, context):
+            return context.resources.io_resource
+
+    @io_manager(required_resource_keys={"io_resource"})
+    def my_io_manager(context):
```

Comment:
```suggestion
    def my_io_manager():
```
Linter complains about this